dev/devicelab/manifest.yaml

```diff
@@ -144,6 +144,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["has-android-device"]
 
+  flutter_view__start_up:
+    description: >
+      Measures the startup time of the Flutter View app on Android.
```

Comment:
Let's reword this to "Verifies that Flutter View can be used from an Android project."